homeassistant/components/imeon_inverter/strings.json

```diff
@@ -0,0 +1,186 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Add Imeon inverter",
+        "data": {
+          "address": "[%key:common::config_flow::data::url%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "address": "The address of the inverter",
+          "username": "Your inverter login",
+          "password": "Your inverter password"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "invalid_host": "[%key:common::config_flow::error::invalid_host%]",
+      "invalid_route": "Unable to request the API, make sure 'API Module' is enabled on your device"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "battery_autonomy": {
+        "name": "Battery autonomy"
+      },
+      "battery_charge_time": {
+        "name": "Battery charge time"
+      },
+      "battery_power": {
+        "name": "Battery power"
+      },
+      "battery_soc": {
+        "name": "Battery SOC"
```

Comment:
you made a comment on why you would use the device class translation for battery, but I don't think SOC makes it clearer. As in, it's quite the if you know you know term. So I would either go for the device class translation or just write it fully